configuration/concepts/newEpisodesOfDiseasesReportConcepts.csv

```diff
@@ -0,0 +1,9 @@

```

Comment:
MSPP New Episodes of Diseases Report - All diagnoses

Reply:
This concept set should be removed if we use directly the Diagnosis Set of Sets (as per comment below)